webv2/web.go

```diff
@@ -1327,6 +1340,8 @@ func getSchemaFile(w http.ResponseWriter, r *http.Request) {
 	schemaFileName := "frontend/" + filePrefix + "schema.txt"
 
 	sessionState := session.GetSessionState()
+	sessionState.Conv.ConvLock.RLock()
+	defer sessionState.Conv.ConvLock.RUnlock()
```

Comment:
If there is already a lock taken above, what is the advantage of taking a Rlock again?

Reply:
It is a part of different API